src/app/zap-templates/common/ClustersHelper.js

```diff
@@ -28,7 +28,7 @@ const zclQuery          = require(zapPath + 'db/query-zcl.js')
 
 const { Deferred }    = require('./Deferred.js');
 const ListHelper      = require('./ListHelper.js');
-const StringHelper    = require('./StringHelper.js');
+const StringHelper    = require(zapPath + 'generator/helper-string-matter.js');
```

Comment:
why was this change made? this seems  to couple chip-tool bidirectionally with zap even more - matter specific code into zap, while matter sdk contains zap code as well.

Reply:
I initially wanted to start looking into moving all the code related to zap back into the zap repo. Since it makes more sense to have all the zap code in one place instead of being scattered into so many places in the CHIP repos. I started with one file but I did not want to club that with the work I was doing so I stopped with just one file. I can revert this change for now but I really feel we should not have zap dependent code outside the zap repo.